app/Services/Backups/InitiateBackupService.php

```diff
@@ -111,6 +115,18 @@ public function handle(Server $server, string $name = null): Backup
             );
         }
 
+        // Check if the server has reached or exceeded it's backup limit
+        if (! $server->backup_limit || $server->backups()->where('is_successful', true)->count() >= $server->backup_limit) {
+            if($override) {
```

Comment:
I try to avoid `if/else` blocks like this, and lean towards a cleaner negation method of:

```
if (!$override) {
    throw X
}

// Do the other thing here
```

This reads a little cleaner to me and makes it a little clearer that there is a single thing that will cause the process to exit.